docs/api/datasets/non_geo_datasets.csv

```diff
@@ -39,7 +39,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `Million-AID`_,C,Google Earth,-,1M,51--73,,0.5--153,RGB
 `MMEarth`_,"C, S","Aster, Sentinel, ERA5","CC-BY-4.0","100K--1M",,"128x128 or 64x64",10,MSI
 `NASA Marine Debris`_,OD,PlanetScope,"Apache-2.0",707,1,256x256,3,RGB
-`OSCD`_,CD,Sentinel-2,"CC-BY-4.0",24,2,"40--1,180",60,MSI
+`OSCD`_,CD,Sentinel-2,"CC-BY-4.0",24,2,"241--1,180",60,MSI
```

Comment:
I believe 40 is the smallest dimension if you look at imgs, 241 is the smallest dimension if you look at imgs_rect (which we use).